react/features/base/avatar/components/web/StatelessAvatar.js

```diff
@@ -38,11 +45,79 @@ type Props = AbstractProps & {
     useCORS?: ?boolean
 };
 
+/**
+ * Creates the styles for the component.
+ *
+ * @param {Object} theme - The current UI theme.
+ *
+ * @returns {Object}
+ */
+const styles = theme => {
+    return {
+        avatar: {
+            backgroundColor: '#AAA',
+            borderRadius: '50%',
+            color: 'rgba(255, 255, 255, 1)',
+            fontWeight: '100',
+            objectFit: 'cover',
+
+            '&.avatar-small': {
+                height: '28px !important',
+                width: '28px !important'
+            },
+
+            '&.avatar-xsmall': {
+                height: '16px !important',
+                width: '16px !important'
+            },
+
+            '& .jitsi-icon': {
+                transform: 'translateY(50%)'
+            },
+
+            '& svg': {
```

Comment:
Please use the `avatar-svg` class here, as just using svg may affect using an icon avatar